Gemfile

```diff
@@ -1,13 +1,15 @@
 source "https://rubygems.org"
 
-gem 'autoproj', git: 'https://github.com/rock-core/autoproj'
+gem 'autoproj', git: 'https://github.com/rock-core/autoproj', branch: 'merged_all_ci_prs'
+gem 'autobuild', git: 'https://github.com/rock-core/autobuild', branch: 'merged_all_ci_prs'
```

Comment:
Are we really merging these to master? I would prefer we merged autoproj and autobuild as well...

Reply:
No, we are not.